content/geometry/ManhattanMST.h

```diff
@@ -0,0 +1,40 @@
+/**
+ * Author: chilli, Takanori MAEHARA
+ * Date: 2019-11-02
+ * License: CC0
+ * Source: https://github.com/spaghetti-source/algorithm/blob/master/geometry/rectilinear_mst.cc
+ * Description: Given N points, returns up to 8*N edges, which are guaranteed
+ * to contain the Manhattan MST. The Manhattan MST of a set of points is the
+ * MST where the weight of the edge between any pair of points is the Manhattan
+ * distance between the points. Edges are in the form (distance, src, dst). Use
```

Comment:
"...guaranteed to contain a minimum spanning tree for the graph with edge weights `w(p, q) = |p.x - q.x| + |p.y - q.y|`."? (the current wording is a bit verbose)